### DISCUSSION TITLE
When moving a project to a new team/account, remind user that any connected Edge Configs will not be transferred

### DISCUSSION THREAD

**Discussion by taravancil** [vercel/community#1206] [Idea]
I was just transferring some projects over to a team account, and I thought it would have been helpful if Vercel gave me a reminder that my Edge Config would not also be transferred with the connected project.

**Comment by swarnava** [comment]
Hi @taravancil , Thank you for the feedback! I will forward this to relevant team to take closer look! :)

**Comment by dferber90** [comment]
Thank you for your idea @taravancil! This is now available :)

<img width="50%" src="https://user-images.githubusercontent.com/1765075/231055643-29bf9174-55d9-4cab-8320-597a5dfdd729.png" />
